lib/ts/ink_args.cc

```diff
@@ -223,7 +223,7 @@ process_args_ex(const AppVersionInfo *appinfo, const ArgumentDescription *argume
       // Deal with long options ...
       for (i = 0; i < n_argument_descriptions; i++) {
         // handle the runroot arg
-        std::string cur_argv = *argv + 2;
+        std::string_view cur_argv = *argv + 2;
```

Comment:
Are we positive this has the same semantics? The std:;string copies (I assume) the string from argv, whereas string_view sill only have a pointer into the argv. Have you made sure this is safe in all places? The possible problem being if argv goes out of scope, but something still has  the string_view "pointer", you are now addressing garbage.

Reply:
There should be no issue with it since it is only used once immediately after this assigning in the following line.

Reply:
ya this is safe